### ISSUE TITLE
Way to disable the store

### ISSUE THREAD

**Issue by avandecreme** [meltingice/CamanJS#185]
In my use case, I am creating and destroying a lot of canvases.
However, every time I call Caman on one of them, it get cached and thus never released by the GC.

Is there a clean way to disable the store?

This is how I am doing it right now:
```javascript
                var camanNoStore = function(canvas, callback) {
                    var storePutBackup = Caman.Store.put;
                    Caman.Store.put = function() {};
                    /* jshint newcap: false */
                    Caman(canvas, function() {
                        callback.bind(this)();
                    });
                    Caman.Store.put = storePutBackup;
                };
```
I can then call camanNoStore as I would call Caman.

**Comment by avandecreme** [comment]
Actually, I realized this hack doesn't work.
So I can't find anything better than setting `Caman.Store.put = function() {};` forever.

**Comment by jocoolie** [comment]
Can't you use `Caman.Store.flush(name)` to remove them? Or does that not work for you?

**Comment by jocoolie** [comment]
If you're up for editing the Caman file, you could probably also use the block of code in `finishInit()` to skip the store step: 

Javascript:

     if (!Caman.NodeJS) {
           Store.put(this.id, this);
        }

Coffeescript (caman.coffee):

    Store.put @id, @ unless Caman.NodeJS

which could be changed to something like

    Store.put @id, @ unless Caman.NodeJS or Caman.DisableStore

And then set DisableStore to true on initialization.

**Comment by avandecreme** [comment]
Yes that would be the best option but I am not sure this project is maintained anymore so I am not willing to make a PR for now.

**Comment by jnccneto** [comment]
Hi
I am facing the same problem. Any good fix for this? didn't want to change the source code. although it seems like a project closed right now
